README.md

```diff
@@ -60,6 +60,30 @@ Protractor flake defaults to using the `standard` parser, which will typically p
 
 You can override this with the `parser` option, specifying one of the [built in parsers](src/parsers/index.js).
 
+#### Cucumberjs parser
```

Comment:
Good suggestion, will do.
As may have read the log of cucumberjs has changed. This has an effect on the existing `cucumber` parser. Do you want `protractor-flake` to be backwards compatible (old versions of cucumber) or shall I add a pear dependency for `cucumberjs` > 0.9 and make the `cucumber`-parser also parse `capability` and `mulitCapabilities` so we have 1 parser called `cucumber`?